nodeList.json

```diff
@@ -0,0 +1,900 @@
+[
```

Comment:
We'd need to add keys to the node objects for to it validate. I'd be okay with keeping it as is for now.